### PR TITLE
Implements "stale issues" policy

### PR DESCRIPTION
**Pull request by Frijol**
Adds config file for the [stale](https://probot.github.io/apps/stale/) probot and documents the policy. Reviewer, please also allow the Probot app access to our repos!

### CODE REVIEW

**Comment by Frijol** [comment]
(once this PR is accepted I will implement it on our other active repos as well)

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. If it should not be closed, please comment! Thank you
```
Comment (Mr0grog) [inline .github/stale.yml]: Can we add the timeframe here? Or better not to suggest to someone that they have time?

**Review by Mr0grog — APPROVED** [review]
Looks good to me! I added one thought inline.

File: .github/stale.yml (hunk 1 above)
Comment (Frijol) [reply .github/stale.yml]: Oh good point! Will add.

**Comment by Frijol** [comment]
@Mr0grog are you able to give Probot permission to access our repos?

**Comment by Mr0grog** [comment]
I got an e-mail to approve it, but when I went to do so yesterday evening, it looked like someone already had.

**Pull request merged**